### PR TITLE
Added History Command

### LINKED ISSUE #88: As a user, I can view the history of food items that I have bought in the past,
...so that I can better track my food habits.

### PR DESCRIPTION
**Pull request by Leeyp**
Closes #88

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,8 +6,29 @@
...
+    * [List expiring foods: expiring](#list-expiring-foods-expiring)
+    * [List categories with food running low: runninglow](#list-categories-with-food-running-low-runninglow)
+    * [Modify the minimum quantity limits: setlimit](#modify-the-minimum-quantity-limits-setlimit)
+    * [List history of items added: history](#list-history-of-items-added-history)
+    * [Clear list history of items added: history clear](#clear-list-history-of-items-added-history-clear)
+    * [Get help message: help](#get-help-message-help)
+    * [Exit the application: bye](#exit-the-application-bye)
+  * [FAQ](#faq)
+  * [Command Summary](#command-summary)
+
```
Comment (kwokyto) [inline docs/UserGuide.md:32]: Can we list the features by the command instead? Like without the description at the front, else our contents page would look kinda message

File: docs/UserGuide.md [hunk 2]
```diff
@@ -6,8 +6,29 @@
 If you can type fast, `FridgeFriend` can track your cold or frozen groceries faster and easier than any other apps.
 It is written in Java, and has more than 3.2kLoC.
 
-* Table of Contents
-  {:toc}
+Table of Contents
```
Comment (kwokyto) [inline docs/UserGuide.md]: Can we just name this "Contents" cos technically theres no table HAHAHAA
Also can we put a  ## heading for this?

File: src/main/java/seedu/fridgefriend/command/HistoryCommand.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import seedu.fridgefriend.utilities.LoggingHandler;
+import seedu.fridgefriend.utilities.Storage;
+import seedu.fridgefriend.utilities.Ui;
+
+/**
+ * Represents a command to read the history data from the history textfile.
+ * History textfile is stored by default in data/historyData.txt.
+ * Food data is automatically added into history textfile,
+ * after every AddCommand is called.
+ * History can also be cleared,
```
Comment (kwokyto) [inline src/main/java/seedu/fridgefriend/command/HistoryCommand.java]: can we put as much into 1 line? i dont think we need a new line for each sentence

File: src/main/java/seedu/fridgefriend/utilities/Storage.java [hunk 4]
```diff
@@ -22,11 +22,19 @@
 import java.nio.file.Paths;
 import java.util.Scanner;
 
+/**
+ * Reads data from,
+ * and writes data to,
+ * the local disk.
+ *
+ */
```
Comment (kwokyto) [inline src/main/java/seedu/fridgefriend/utilities/Storage.java:28]: Cann we put all this into 1 line

**Review by kwokyto — APPROVED** [review]
Other than the comments, LGTM

File: src/main/java/seedu/fridgefriend/command/HistoryCommand.java (hunk 3 above)
Comment (Leeyp) [reply src/main/java/seedu/fridgefriend/command/HistoryCommand.java]: This one abit long so i think cannot tho unless i cut

File: docs/UserGuide.md (hunk 2 above)
Comment (Leeyp) [reply docs/UserGuide.md]: alright

File: docs/UserGuide.md (hunk 1 above)
Comment (Leeyp) [reply docs/UserGuide.md:32]: alright

**Comment by Leeyp** [comment]
Made changes, requesting approval from @Vinci-Hu and @SimJJ96  and @joohwan58

File: src/main/java/seedu/fridgefriend/command/HistoryCommand.java (hunk 3 above)
Comment (Leeyp) [reply src/main/java/seedu/fridgefriend/command/HistoryCommand.java]: I condensed a bit but into 1 single line would violate code quality ("Line too long!")

File: docs/UserGuide.md [hunk 5]
```diff
@@ -6,8 +6,28 @@
 If you can type fast, `FridgeFriend` can track your cold or frozen groceries faster and easier than any other apps.
 It is written in Java, and has more than 3.2kLoC.
 
-* Table of Contents
-  {:toc}
+## Contents
+
+* [User Guide](#user-guide)
```
Comment (kwokyto) [inline docs/UserGuide.md]: can we remove this bullet point? then we can shift everything below one tab to the left

Comment (Leeyp) [reply docs/UserGuide.md]: Sure, implemented.

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**